src/main/java/uk/gov/mint/Handler.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.mint;
+
+public interface Handler {
+    public abstract void handle(String payload);
```

Comment:
can we please remove `public abstract` from the signature? it gives me warnings in intellij.